lib/logstash/outputs/datadog_metrics.rb

```diff
@@ -116,8 +107,13 @@ def flush(events, final=false)
   end # def flush
 
   private
+
+  def series_to_json(series)
```

Comment:
I wonder what is the benefit of having this function? if only used once? could we not just inline it? I guess the compile will do, but .... :smiley_cat: